client/commonFramework/show-completion.js

```diff
@@ -47,13 +47,23 @@ window.common = (function(global) {
           next();
         });
 
+      let timezone = 'UTC';
+      try {
+        timezone = moment.tz.guess();
```

Comment:
No, the error happens within timezone. The assignment will not take affect until guess() resolves but since it throws it never resolves. It is caught by the catch below.